### PR TITLE
add rule for Futurezone.at

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/FutureZone.at.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	Time Out:
+		- c1
+		- c2
+-->
+<ruleset name="FutureZone.at)">
+
+	<target host="api.futurezone.at" />
+	<target host="community.futurezone.at" />
+	<target host="futurezone.at" />
```
Comment (J0WI) [inline src/chrome/content/rules/FutureZone.at.xml]: Please put the ^ target always at first.

File: src/chrome/content/rules/FutureZone.at.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		- images05
+		- link.nl
+		- mails
+		- reserve
+
+	Time Out:
+		- c1
+		- c2
+-->
+<ruleset name="FutureZone.at)">
```
Comment (J0WI) [inline src/chrome/content/rules/FutureZone.at.xml]: That bracket looks wrong ;)

File: src/chrome/content/rules/FutureZone.at.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	<target host="www.futurezone.at" />
+
+	<securecookie host="." name="." />
+
+	<rule from="^http:"
+		to="https:" />
+
+</ruleset>
+
+
```
Comment (J0WI) [inline src/chrome/content/rules/FutureZone.at.xml]: Could you also remove this line breaks?

**Comment by J0WI** [comment]
The rule itself LGTM.

**Comment by numismatika** [comment]
Brackets and linebreaks  removed, targets sorted, rebased to master

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**